sdk/storage/azure-storage-blobs/test/perf/src/main.cpp

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <azure/perf.hpp>
+
+#include "azure/storage/blobs/test/perf/download_blob.hpp"
+
+int main(int argc, char** argv)
+{
+
+  // Create the test list
+  std::vector<Azure::Perf::TestMetadata> tests{
+      Azure::Storage::Blobs::Test::Performance::DownloadBlob::GetTestMetadata()};
```

Comment:
There's no `Test` namespace under `Blobs`. We only have one `Test` namespace, so `Azure::Storage::Test::Performance`

Reply:
On a second thought, I don't think `Performance` is necessary at all. We don't have `Azure::Storage::Test::Ut`, right?

perf test and ut test are two separate executables. We don't mix the source files, so there's no need to distinguish them with different namespaces.